manual/src/cmds/unified-options.etex

```diff
@@ -116,6 +116,11 @@ Pass the given option to the C compiler and linker.
 directory \var{dir}. \comp{(See the "-custom" option.)}
 }%notop
 
+\notop{%
+\item["-cmi-file" \var{filename}]
+Use the given interface file to type-check the ML source file to compile.
```

Comment:
It would be nice to indicate what's the behaviour if the option is unspecified, i.e. IIRC that the `.cmi` will be looked up in the load path but that it is expected that there is an existing `.mli` file at the same place as the .ml source.

Reply:
more like: **if** there is an `.mli` a the same place as the `.ml`, **then** the `.cmi` will be looked up in the load path, **else** the `.cmi` will be output by the compiler (next to the `.cmo` or `.cmx`).